components/appointmentPicker/appointmentPicker.php

```diff
@@ -0,0 +1,78 @@
+<style>
```

Comment:
@CollinsSpencer Should these inline styles be moved to the appointmentPicker.css?